frontend/src/components/dashboard/dashboard-details.tsx

```diff
@@ -0,0 +1,367 @@
+import React from "react";
+import { connect } from "react-redux";
+import { withStyles } from "tss-react/mui";
+// Components
+import { Container, Grid, Paper } from "@mui/material";
+// Tables
+import ClusterDashboardTableStyled from "./clusters-dashboard-table";
+import AgentDashboardTableStyled from "./agents-dashboard-table";
+import EntriesDashBoardTableStyled from "./entries-dashboard-table";
+import DashboardDrawerStyled from "./dashboard-drawer";
+import { RootState } from 'redux/reducers';
+import { Theme } from '@mui/material/styles'; 
+const styles = (theme:Theme)  => ({
+  root: { 
```

Comment:
Two of your TS files have double space indentation while the other two have four space indentation. I also seem some inconsistencies with the style. I think eventually we should have a code linter to check for this but just keep this in mind.